modules/http-statistics-monitoring/pom.xml

```diff
@@ -29,17 +29,6 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-jar-plugin</artifactId>
```

Comment:
Sorry, this was a mistake I made when I intended to remove the maven-source-plugin. Fixed in next commit and maven-source-plugin is removed.